dlgr/griduniverse/experiment.py

```diff
@@ -303,6 +307,19 @@ def __init__(self, **kwargs):
             self.color_costs = [2**i for i in range(self.num_colors)]
             random.shuffle(self.color_costs)
 
+        # get food spawning probability distribution function and args
+        self.probability_function_args = []
+        parts = self.food_probability_distribution.split()
+        if len(parts) > 1:
+            self.food_probability_distribution = parts[0]
```

Comment:
We need some error catching for incorrect keys entered as probability distributions.

Reply:
Right now, if you enter an invalid distribution name, it just is ignored, and simple random is used instead (see lines 318-320 below). No exception is raised, so the experiment always runs. How would you prefer to handle invalid names?

Reply:
I think maybe it should return a key value error, just incase somebody misspells a distribution like "sinusoidal" and it ends up running the random distribution.

Reply:
The only problem is that the error will only be visible in the heroku logs and will result in an empty grid displayed when the experiment starts.

Reply:
Hmm, maybe some log saying that it's an incorrect key and runs with random anyway? Just so there's something in the logs.

Reply:
Added logging.